modules/ROOT/pages/mutations/delete.adoc

```diff
@@ -51,13 +55,12 @@ mutation {
 }
 ----
 
-This will delete the post using the autogenerated ID that would have been returned after that post's creation.
+This should delete the post using the autogenerated ID that was returned after that post's creation.
+Consequently, `nodesDeleted` should be equal `1` (the post) and `relationshipsDeleted` should also equal `1` as the `HAS_POST` relationship between the `Post` and its author was deleted.
 
-`nodesDeleted` would equal 1 (the post) and `relationshipsDeleted` would also equal equal 1 (the `HAS_POST` relationship between the Post and its author).
+== Nested `delete`
 
-== Nested Delete
-
-Say that if when you delete a User, you want to delete _all_ of their Posts as well. This can be achieved using a single nested delete operations:
+In case you want to delete a `User` *and* all of their posts, you can use a single nested `delete` mutation:
```

Comment:
I can't believe how bad this documentation is (by us, not you) - this isn't even accurate!!!

You can achieve this using:

```gql
mutation {
  deleteUsers(where: { name: "Jane Doe" }, delete: { posts: {} }) {
    nodesDeleted
    relationshipsDeleted
  }
}
```

Note that I didn't even need the empty `where`, which makes the majority of the text below this redundant!